substrait_consumer/tests/functional/extension_functions/approximation_snapshots/function_test_results/approx_count_distinct_result.txt

```diff
@@ -1,2 +1,2 @@
-523942
+658483
```

Comment:
This seems like something that shouldn't change.

Reply:
I would say the opposite: since the function is "approximate," we shouldn't be testing for equality.

It is true, though, that we might require it to be deterministic, so that value shouldn't change *for the same version of the the same consumer* (TBD). But we haven't recorded the version of the consumer until now, so that may very well explain the change...